src/plugins/vis_type_xy/public/utils/render_all_series.tsx

```diff
@@ -105,13 +107,13 @@ export const renderAllSeries = (
               id={id}
               name={getSeriesName}
               color={getSeriesColor}
-              tickFormat={yAspect.formatter}
+              tickFormat={yAspects[0].formatter}
```

Comment:
I assume the `formatter` would be the same across these aspects. Is that actually true?

Reply:
Yes, I have assumed the same, if I understand percentiles correctly, the formatter should be the same for each calculated value.